src/vtbackend/Functions.h

```diff
@@ -618,10 +623,82 @@ inline auto const& functions() noexcept
     return funcs;
 }
 
+class SupportedSequences
+{
+  public:
+    SupportedSequences()
+    {
+        for (auto [index, seq]: crispy::indexed(functions()))
+        {
+            _supportedSequences[index] = seq;
+        }
+        crispy::sort(
+            _supportedSequences,
+            [](FunctionDefinition const& a, FunctionDefinition const& b) constexpr { return compare(a, b); });
+        _lastIndex = _supportedSequences.size();
+    }
+
+    [[nodiscard]] gsl::span<FunctionDefinition const> getAllSequences() const noexcept
```

Comment:
isn't that function basically the same as the original `functions()` one, except that now it's not sorted across the full range anymore? We should probably return simply the result of `functions` here. 
But then we could simply make this function here static, and move the implementation of `functions()` into this function, and get rid of the other all together.

Reply:
Based on my comment below (which I wrote first, sorry), we could then also get rid of the get-prefix and all this `allSequences()`